integration-tests/contracts/ethereum_contracts.go

```diff
@@ -99,11 +99,6 @@ func DefaultOffChainAggregatorOptions() OffchainOptions {
 
 // DefaultOffChainAggregatorConfig returns some base defaults for configuring an OCR contract
 func DefaultOffChainAggregatorConfig(numberNodes int) OffChainAggregatorConfig {
-	if numberNodes <= 4 {
-		log.Err(fmt.Errorf("insufficient number of nodes (%d) supplied for OCR, need at least 5", numberNodes)).
-			Int("Number Chainlink Nodes", numberNodes).
-			Msg("You likely need more chainlink nodes to properly configure OCR, try 5 or more.")
-	}
```

Comment:
why are we removing it?

Reply:
We count only worker nodes here, I can run it with 1+4